Formula/ipython@5.rb

```diff
@@ -99,10 +143,50 @@ class IpythonAT5 < Formula
   end
 
   def install
-    virtualenv_install_with_resources
+    xy = Language::Python.major_minor_version "python"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python#{xy}/site-packages"
+
+    # install other resources
+    ipykernel = resource("ipykernel")
+    (resources - [ipykernel]).each do |r|
+      r.stage do
+        system "python", *Language::Python.setup_install_args(libexec/"vendor")
+      end
+    end
+
+    # install and link IPython
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python#{xy}/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+    bin.install libexec/"bin/ipython"
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+
+    # install IPython man page
+    man1.mkpath
+    man1.install libexec/"share/man/man1/ipython.1"
+
+    # install IPyKernel
+    ipykernel.stage do
+      system "python", *Language::Python.setup_install_args(libexec/"vendor")
+    end
+
+    # install kernel
+    system libexec/"bin/ipython", "kernel", "install", "--prefix", share
+    inreplace share/"share/jupyter/kernels/python2/kernel.json", "]", <<-EOS.undent
+      ],
+      "env": {
+        "PYTHONPATH": "#{ENV["PYTHONPATH"]}"
+      }
+    EOS
+  end
+
+  def post_install
+    (etc/"jupyter/kernels/python2").install Dir[share/"share/jupyter/kernels/python2/*"]
   end
 
   test do
     assert_equal "4", shell_output("#{bin}/ipython -c 'print 2+2'").chomp
+
+    system bin/"ipython", "kernel", "install", "--prefix", testpath
+    assert File.exist?(testpath/"share/jupyter/kernels/python2/kernel.json"), "Failed to install kernel"
```

Comment:
You can use `assert_predicate` like this: https://github.com/Homebrew/homebrew-core/blob/master/Formula/bento4.rb#L26